src/v0/destinations/campaign_manager/data/CampaignManagerTrackConfig.json

```diff
@@ -12,11 +12,7 @@
   {
     "destKey": "timestampMicros",
     "sourceKeys": "timestamp",
-    "sourceFromGenericMap": true,
-    "required": true,
```

Comment:
Is it not mandatory parameter ?

Reply:
Since this field will always be present, so removed this parameter.